### ISSUE TITLE
Loinc 26465-5:  "Leukocytes [#/volume] in Cerebral spinal fluid"

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4017]
Suggest creating a new term for Loinc: 26465-5 [Leukocytes [#/volume] in Cerebral spinal fluid] 
New term label:
Abnormal leukocyte count in cerebral spinal fluid
Increased leukocyte count in cerebral spinal fluid
Decreased leukocyte count in cerebral spinal fluid
Parent: 'Abnormal leukocyte morphology'
Your biocurator ID for loinc2hpo (if desired): HPO:nvasilevsky

**Comment by nicolevasilevsky** [comment]
also need these terms in pleural fluid, peritoneal fluid, body fluid, synovial fluid

**Comment by pnrobinson** [comment]
this is prob CSF pleocytosis

**Comment by nicolevasilevsky** [comment]
add syn to CSF pleocytosis

**Comment by nicolevasilevsky** [comment]
create new term for abn peritoneal fluid- maybe child of ascites, then add abn leukocyte count in peritoneal fluid as a child

**Comment by pnrobinson** [comment]
This item is https://hpo.jax.org/app/browse/term/HP:0012229  @nicolevasilevsky 
Pls add synonym "Increased leukocyte count in CSF"
